third_party_integration/huggingface_transformers/BootstrapNAS_HF/0001-Modifications-for-BootstrapNAS-usage.patch

```diff
@@ -0,0 +1,1582 @@
+From 193c9e67e6a25a10ac4af770b4b491c2859d7907 Mon Sep 17 00:00:00 2001
+From: "J. Pablo Munoz" <pablo.munoz@intel.com>
+Date: Wed, 30 Nov 2022 17:00:14 -0800
+Subject: [PATCH] Enable BootstrapNAS
+
+---
+ examples/pytorch/language-modeling/run_clm.py |  87 ++++++--
+ examples/pytorch/question-answering/run_qa.py |  72 ++++++-
+ .../pytorch/question-answering/trainer_qa.py  |   5 +-
+ .../pytorch/text-classification/run_glue.py   |  88 +++++++--
+ .../pytorch/text-classification/run_xnli.py   |  74 ++++++-
+ .../pytorch/token-classification/run_ner.py   |  90 +++++++--
+ .../nncf_bootstrapnas_bert_config_conll.json  |  65 ++++++
```

Comment:
Do these configs and PR in general allow achieving some results or intended for demonstration purposes?

Reply:
Demonstration purposes. Thanks!